src/Wizard/WizardContent/WizardContent.tsx

```diff
@@ -3,9 +3,13 @@ import * as React from "react";
 import { Box } from "@mui/material";
 import { WizardContentProps } from "./WizardContent.types";
 
-export default function WizardContent({ children }: WizardContentProps) {
+export default React.forwardRef<HTMLDivElement, WizardContentProps>(
+  { children }: WizardContentProps,
+  ref
+) {
```

Comment:
I seem to get an eslint error:

![image](https://github.com/IPG-Automotive-UK/react-ui/assets/143519265/28b857d7-da3f-4ed7-b970-1069d7482d02)

The suggestion fixes it:

```suggestion
export default React.forwardRef<HTMLDivElement, WizardContentProps>(
  function WizardContent({ children }, ref) {
```